### ISSUE TITLE
Docker compilation file for centos6

### ISSUE THREAD

**Issue by kramasamy** [twitter/heron#149]
It will be nice if we could add a centos6 docker file as well - this can be used to release centos binaries. Centos 7 is the latest for which we have already a docker file. Heron release should atleast one version lower than the latest release so that people have time to catch up.

**Comment by kramasamy** [comment]
@windie - I was wondering if you are interested in this.

**Comment by kramasamy** [comment]
@windie - yes he picked it up. Perhaps, you could help testing this. I am testing this in a Mac.

**Comment by bclicn** [comment]
Could you also release installers for CentOS6? Since most of our servers are running on CentOS 6.5.
Thank you!

**Comment by Code0x58** [comment]
probably fair to close this, as now 7+8 are the latest, then there's also the intent to move towards a limited set of supported docker bases which can be left to resolve what should/shouldn't exist

**Comment by nwangtw** [comment]
Yeah. I think we have had a conclusion after our discussion. Let's move it
to a separate folder for reference only without active maintenance.

**Comment by joshfischer1108** [comment]
Have we decided on folder structure on the mailing list yet?
